docs/eigenda/integrations-guides/dispersal/dispersal-api/overview.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+
+The EigenDA Disperser is a permissioned, untrusted service hosted by Eigen Labs which serves as a coordinator between clients, the operator set, and onchain Ethereum contracts for the purpose of dispersing __and retriving__ blobs. In the future the EigenDA disperser will be made permissionless, so that clients can interact with the EigenDA network and contracts directly to avoid a liveness dependency on the Eigen Labs disperser.
```

Comment:
I think we should try to reserve color commentary for the architecture overview (to be written). I think a more neutral way to provide the needed information might be something like: 

`The EigenDA disperser provides an API for dispersing and retrieving blobs to and from the EigenDA network in an untrusted fashion. (Note: as part of its essential data availability guarantee, the EigenDA network already supports direct communication with the EigenDA network for blob retrieval; permissionless dispersal of blobs to the EigenDA network is planned as a future protocol upgrade).`